MDANSE/Src/MDANSE/Framework/Jobs/GaussianDynamicIncoherentStructureFactor.py

```diff
@@ -25,8 +25,12 @@
 
 
 class GaussianDynamicIncoherentStructureFactor(IJob):
-    """
-    Computes the dynamic incoherent structure factor S_inc(Q,w) for a set of atoms in the Gaussian approximation.
+    """Computes the dynamic incoherent structure factor in the Gaussian approximation.
+
+    Gaussian approximation is exact for a system of free particles and a system of
+    particles undergoing brownian motion. The results of this analysis will be close
+    to the Dynamic Incoherent Structure Factor analysis in the limits of very
+    short Q and very long Q, and will differ from it for intermediate Q values.
```

Comment:
```suggestion
    short :math:`\mathbf{q}` and very long :math:`\mathbf{q}`, and will differ from it for intermediate :math:`\mathbf{q}` values.
```

Update maths to used math font.